superset/connectors/sqla/models.py

```diff
@@ -255,13 +255,19 @@ class TimePeriod(enum.Enum):
     day = 60 * 24
 
 
+class ExecutionType(enum.Enum):
```

Comment:
Are we set on using an enum type here? The only reason I ask is because it's sort of pain to serialize when sending the option back to the front end. Enum classes aren't serializable, so you have to extract the attribute and serialize that instead.

Reply:
I added a dictionary type with the same data